PENDING.md

```diff
@@ -17,6 +17,8 @@ BREAKING CHANGES
     utilize a validator's operator address must now use the new Bech32 prefix,
     `cosmosvaloper`.
     * [cli] [\#2190](https://github.com/cosmos/cosmos-sdk/issues/2190) `gaiacli init --gen-txs` is now `gaiacli init --with-txs` to reduce confusion
+    * [cli] \#2073 --from can now be either an address or a key name
```

Comment:
@mslipper can you update this to reflect the new standard? You can just run the `linkify_changelog.py` script.

Reply:
linkify doesn't need to be ran on each PR. We can just have the release coordinator run it. (Theres no advantage to doing it every PR versus only doing it once. It also would mask the changelog update here as it updates other lines as well)